src/app/modules/shared-components/components/score-ring/score-ring.component.html

```diff
@@ -32,8 +32,10 @@
       fill-opacity="0.4"
```

Comment:
On [this page] as usual user, score rings are a bit truncated on firefox:
![image](https://user-images.githubusercontent.com/16257159/143915865-b2ca1f94-8929-4d76-b0fb-f8325d8b705a.png)

Reply:
I've add some fix, could you test pls, because I have another behaviour on my Firefox

Reply:
Works fine :+1: